samples/zigbee/cli/README.rst

```diff
@@ -0,0 +1,153 @@
+.. _zigbee_cli_sample:
+
+Zigbee: CLI
+###########
+
+The Zigbee CLI sample demonstrates a Zigbee router (with the possibility of being a coordinator) that uses the Command Line Interface inside Zephyr's :ref:`zephyr:shell_api` for interaction.
+
+The sample can be used for several purposes, including:
+
+* Initial configuration of the network - forming a network as a coordinator, adding devices to the network with the install codes, setting the extended PAN ID
+* Troubleshooting the existing network - reading and writing attributes, subscribing to the reports of the parameters
+* Benchmarking - measuring time needed for message to travel from one node to another
+
+Available CLI interfaces
+************************
+
+The CLI can be accessed by using the following interfaces:
+
+* UART
+* RTT
+
+These interfaces are completely independent one from another and can be used simultaneously or exclusively.
+
+The sample demonstrates the usage of commands listed in `Zigbee CLI Reference`_.
+Zigbee CLI is integrated into the system shell accessible over serial connection.
```

Comment:
Yes, by default, CLI commands are accessible by UART. Should it mention other interfaces here?

Reply:
Discussed f2f.